packages/admin/admin/src/common/buttons/feedback/FeedbackButton.tsx

```diff
@@ -31,24 +31,41 @@ const Tooltip = createComponentSlot(CometTooltip)<FeedbackButtonClassKey>({
     slotName: "tooltip",
 })();
 
-export interface FeedbackButtonProps
+type FeedbackStateControl = "props" | "onClickPromise";
+
+export interface FeedbackButtonPropsBase
     extends ThemedComponentBaseProps<{
             root: typeof LoadingButton;
             tooltip: typeof CometTooltip;
         }>,
-        LoadingButtonProps {
-    loading?: boolean;
-    hasErrors?: boolean;
+        Omit<LoadingButtonProps, "loading"> {
+    feedbackStateControl: FeedbackStateControl;
```

Comment:
This would need to be optional to not be a breaking change

Reply:
https://github.com/vivid-planet/comet/pull/2237/commits/34316b6c490e080772be0de5d9012e241d7b67a0